### ISSUE TITLE
To add “暗干” and "暗神" 

### ISSUE THREAD

**Issue by P-Rainbow** [kentang2017/kinqimen#12]
Hi Mr. Tang,

It's been some time. Hope all's well with you. :)

Would you consider to add “暗干” and "暗神" to each "宫"？

**Comment by kentang2017** [comment]
yes will consider to add those information. right now i am writing kintaiyi

**Comment by P-Rainbow** [comment]
@kentang2017 wow! thanks for the update. Kintaiyi will be another remarkable work to the world!
